_pages/introduction-developing-skills.md

```diff
@@ -91,7 +91,7 @@ Now, we'll made a new repository for your **Skill**. The new repository has to f
 Copy the Template Skill into a new directory. Here, we've called the new Skill `skill-training`, but your **Skill** will have a different name.
```

Comment:
Since you are fixing this anyway, want to fix this line?  It says, "we've called the new Skill `skill-training`" but you can clearly see in the code sample below it was called skill-helloworldls (and now skill-hello-world in your PR).  I figured you could edit it since you already have a PR setup.  If not, I can and submit my own.